site/docs/view/size.md

```diff
@@ -54,6 +54,24 @@ For example, the following plot use `21` as a default height.
 
 <span class="vl-example" data-name="bar_1d_rangestep_config"></span>
 
+### Autosize
+
+The specified dimensions of a chart as explained above set the size of the data rectangle (plotting) dimensions. You can override this behavior by setting the autosize property in [the top level specification](spec.html#top-level-specifications).
```

Comment:
- [ ] Please add
  - the `autosize` property
  - , which can be either a string or an autosize parameter object.

- [ ] Missing description about what none, pad, and fit mean.  (Better copy content from https://vega.github.io/vega/docs/specification/#autosize).  

> The specified dimensions of a chart as explained above set the size of the data rectangle (plotting) dimensions.

I think this is an important point that is kinda hidden in this section.  Maybe add this to the intro so we provide better overview how size works?

Reply:
Done